distribution/openrct2.d.ts

```diff
@@ -4398,6 +4397,25 @@ declare global {
         canSelect: boolean;
     }
 
+    interface ProgressBarWidget extends WidgetBase {
+        colour: number;
+        /**
+         * The percentage of the bar to fill, ranging from 0-100 (inclusive).
+         */
+        percentage: number;
+        /**
+         * Used to specify a range of values between which the bar will start blinking.
+         * The bar will blink when lowerBlinkBound <= percentage <= upperBlinkBound.
+         * Set both upper and lower bounds to the same value to disable blinking.
+         */
+        lowerBlinkBound: number;
```

Comment:
@Sadret how about this for the description?

Reply:
@ltsSmitty Wouldn’t it be better to make it blink below the lower bound or above the upper bound? I imagine people would normally want the bar to blink either when the value is too low or too high. And if the higher bound is set lower than the lower bound, the bar can blink if the value is in between.

Reply:
@ltsSmitty This description seems fine.

@mrmagic2020 I agree that it would make more sense; I can imagine more use cases where you want it to blink when the value is very low or very high than cases where you want it to blink in the middle but not at the extreme cases. The problem is that the progress bar widget does not support it, so we would have to rewrite that one, too. That's possible and actually probably not too much work.